perf/renaissance-jmh/playlist.xml

```diff
@@ -0,0 +1,293 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
+	<include>../perf.mk</include>
+	<test>
+		<testCaseName>renaissance-jmh-AkkaUct</testCaseName>
+		<command>
```

Comment:
Do we have the options and the why of those options documented somewhere?

Reply:
FYI - the GC logging is a little short in terms of what Censum needs if that was a goal

Reply:
GC logging was intended to be as simple as possible initially. I can update it with something more substantial, e.g., 

-Xlog:gc*,gc+ref=debug,gc+phases=debug,gc+age=trace,gc+ergo*=trace,safepoint:file=gc.log

Thoughts on whether that's overkill or not?

Reply:
As for the JVM and benchmark options, I'll let @dhanalla follow up this PR with more commentary on the experimentation that was done to configure them.

Reply:
We are using the JMH GC profiling option (-prof gc) in our experiment runs to capture GC allocation data in a JSON file. This enables us to compare the control and treatment results and measure any improvements using python scripts.

Reply:
@dhanalla - Have you taken a look at the Censum tool that our team owns?  Happy to walk you through it and see if it is more useful (or not).

@brianjstafford that's the right set yeah (assuming we use Censum here)